tests/bsim/bluetooth/ll/cis/src/main.c

```diff
@@ -0,0 +1,801 @@
+	}
+	printk("disconnected from peer device.\n");
+
+	if (expected_seq_num < SEQ_NUM_MAX) {
+		FAIL("ISO Data reception incomplete %u (%u).\n",
+		     expected_seq_num, SEQ_NUM_MAX);
+		return;
+	}
+
+	PASS("Peripheral1 ISO tests Passed\n");
+}
+
+static void test_cis_peripheral2(void)
+{
+	test_cis_peripheral1();
+}
+
+static void test_cis_peripheral3(void)
+{
+	test_cis_peripheral1();
+}
+
+static void test_cis_peripheral4(void)
+{
+	test_cis_peripheral1();
+}
+
+static void test_cis_peripheral5(void)
+{
+	test_cis_peripheral1();
+}
+
+static void test_cis_peripheral6(void)
+{
+	test_cis_peripheral1();
+}
+
+static void test_cis_peripheral7(void)
+{
+	test_cis_peripheral1();
+}
+
+static void test_cis_peripheral8(void)
+{
+	test_cis_peripheral1();
+}
+
+static void test_cis_peripheral9(void)
+{
+	test_cis_peripheral1();
+}
+
+static void test_cis_init(void)
+{
+	bst_ticker_set_next_tick_absolute(60e6);
+	bst_result = In_progress;
+}
+
+static void test_cis_tick(bs_time_t HW_device_time)
+{
+	if (bst_result != Passed) {
+		FAIL("test failed (not passed after seconds)\n");
+	}
+}
+
+static const struct bst_test_instance test_def[] = {
+	{
+		.test_id = "central",
+		.test_descr = "Central ISO",
+		.test_post_init_f = test_cis_init,
+		.test_tick_f = test_cis_tick,
+		.test_main_f = test_cis_central,
+	},
+	{
+		.test_id = "peripheral1",
+		.test_descr = "Peripheral ISO 1",
+		.test_post_init_f = test_cis_init,
+		.test_tick_f = test_cis_tick,
+		.test_main_f = test_cis_peripheral1,
+	},
+	{
+		.test_id = "peripheral2",
+		.test_descr = "Peripheral ISO 2",
+		.test_post_init_f = test_cis_init,
+		.test_tick_f = test_cis_tick,
+		.test_main_f = test_cis_peripheral2,
+	},
+	{
+		.test_id = "peripheral3",
+		.test_descr = "Peripheral ISO 3",
+		.test_post_init_f = test_cis_init,
+		.test_tick_f = test_cis_tick,
+		.test_main_f = test_cis_peripheral3,
+	},
+	{
+		.test_id = "peripheral4",
+		.test_descr = "Peripheral ISO 4",
+		.test_post_init_f = test_cis_init,
+		.test_tick_f = test_cis_tick,
+		.test_main_f = test_cis_peripheral4,
+	},
+	{
+		.test_id = "peripheral5",
+		.test_descr = "Peripheral ISO 5",
+		.test_post_init_f = test_cis_init,
+		.test_tick_f = test_cis_tick,
+		.test_main_f = test_cis_peripheral5,
+	},
+	{
+		.test_id = "peripheral6",
+		.test_descr = "Peripheral ISO 6",
+		.test_post_init_f = test_cis_init,
+		.test_tick_f = test_cis_tick,
+		.test_main_f = test_cis_peripheral6,
+	},
+	{
+		.test_id = "peripheral7",
+		.test_descr = "Peripheral ISO 7",
+		.test_post_init_f = test_cis_init,
+		.test_tick_f = test_cis_tick,
+		.test_main_f = test_cis_peripheral7,
+	},
+	{
+		.test_id = "peripheral8",
+		.test_descr = "Peripheral ISO 8",
+		.test_post_init_f = test_cis_init,
+		.test_tick_f = test_cis_tick,
+		.test_main_f = test_cis_peripheral8,
+	},
+	{
+		.test_id = "peripheral9",
+		.test_descr = "Peripheral ISO 9",
+		.test_post_init_f = test_cis_init,
+		.test_tick_f = test_cis_tick,
+		.test_main_f = test_cis_peripheral9,
+	},
```

Comment:
I don't see any reason to define multiple `test_id`s for the peripheral, as they just call the same function anyhow. You can have multiple devices run the same `test_id`. 

If you want to have anything specific for them based on their number (1, 2, 3, ...) you could also use test arguments instead. 

So I suggest to only define a single `peripheral` test ID, and just use that for all the peripherals.